mobile/android/android-application-manager.ts

```diff
@@ -102,4 +127,12 @@ export class AndroidApplicationManager extends ApplicationManagerBase {
 
 		return applicationViews;
 	}
+
+	private getFullyQualifiedActivityRegex(): RegExp {
+		const androidPackageName = "([A-Za-z]{1}[A-Za-z\\d_]*\\.)*[A-Za-z][A-Za-z\\d_]*";
+		const packageActivitySeparator = "\\/";
+		const fullJavaClassName = "([a-z][a-z_0-9]*\\.)*[A-Z_$]($[A-Z_$]|[$_\\w_])*";
+
+		return new RegExp(`${androidPackageName}${packageActivitySeparator}${fullJavaClassName}`, `m`);
```

Comment:
Will put `@cache` as you suggested, thank you!
I thought about using the app identifier for the first part of the regex, but I prefer to keep the general implementation since the app identifier is sometimes overridden by the user in `app.gradle`.